addons/website_slides/views/website_slides_templates_lesson.xml

```diff
@@ -98,15 +98,17 @@
 
 <!-- Slide sub-template: display an item in a list of related slides (Related, Most Viewed, ...) -->
 <template id="slide_aside_card" name="Related Slide">
-    <a class="list-group-item list-group-item-action d-flex align-items-start px-2" t-att-href="'/slides/slide/%s' % (slug(aside_slide))">
-        <div class="me-1 border o_wslides_background_image_aside_card" t-attf-style="background-image: url(#{website.image_url(aside_slide, 'image_256')});"/>
-        <div class="overflow-hidden d-flex flex-column justify-content-start">
-            <h6 t-esc="aside_slide.name" class="o_wslides_desc_truncate_2 mb-1" style="line-height: 1.15"/>
-            <small class="text-600">
-                <t t-esc="aside_slide.total_views"/> Views &#8226; <timeago class="timeago" t-att-datetime="aside_slide.create_date"></timeago>
-            </small>
-        </div>
-    </a>
+    <li>
+        <a class="list-group-item list-group-item-action d-flex align-items-start px-2" t-att-href="'/slides/slide/%s' % (slug(aside_slide))">
```

Comment:
Was the `a` directly inside the `ul`? :open_mouth: This one, I'd prefer it to be in its own **PR**. Probably to send it to the related team, too risky to merge by me in 17.0.

Reply:
> This one, I'd prefer it to be in its own PR

Here it is: https://github.com/odoo/odoo/pull/142116